packages/core/admin/_internal/node/develop.ts

```diff
@@ -24,6 +27,40 @@ interface DevelopOptions extends CLIContext {
   watchAdmin?: boolean;
 }
 
+const cleanupDistDirectory = async ({
+  tsconfig,
+  logger,
+  timer,
+}: Pick<DevelopOptions, 'tsconfig' | 'logger'> & { timer: TimeMeasurer }) => {
+  const distDir = tsconfig?.config?.options?.outDir;
+
+  if (!distDir || !(await fs.pathExists(distDir))) {
+    return;
+  }
+  const timerName = 'cleaningDist' + Date.now();
+  timer.start(timerName);
+  const cleaningSpinner = logger.spinner(`Cleaning dist dir ${distDir}`).start();
+
+  try {
+    const dirContent = await fs.readdir(distDir);
+    const validFilenames = dirContent
+      // Ignore the admin build folder
+      .filter((filename) => filename !== 'build');
```

Comment:
Is there a better way to get the build folder? How do we know someone's not changed it in their webpack config? 🤔

I would assume if you change it there, you need to change it in the strapi set up – if it's not possible to do that we can ignore my comment.

Reply:
I'll throw the question back to you, is there a better way to get it from their webpack config? Do we already have any utilities for it or would we need to write something? Using 'build' will at least get us back to where we were two weeks ago, but I agree it's not ideal.

Reply:
hahaha well upon further reflection we hard code the folder `build` so i think you're fine. Good game of tennis